src/main/java/com/przesylki/parcel/api/letter/getAll/response/LetterResponse.java

```diff
@@ -0,0 +1,70 @@
+package com.przesylki.parcel.api.letter.getAll.response;
+
+import com.przesylki.parcel.dao.entity.*;
+
+public class LetterResponse {
+
+    private Sender sender;
+    private SenderAddress senderAddress;
+    private Receiver receiver;
+    private ReceiverAddress receiverAddress;
+    private CreatedAt createdAt;
```

Comment:
Typ CreatedAt? Rozumiem, że to jest data - więc typ może być Date.
private Date createdAt;
To samo z updatedAt;